core/internal/stream/sender.go

```diff
@@ -761,7 +761,7 @@ func (s *Sender) finishRunSync() {
 
 	// Prevent any new work from being added.
 	//
-	// Note that any work queued up at this point does get processed.
+	// Note that any work queued up at this point still gets processed.
```

Comment:
Do you want to add a note that it can only be asynchronous requests from the py client, like you said?